frontend/src/General/DynamicBreadcrumb.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+    Box,
+    Breadcrumbs,
+    Typography,
+    Link
+} from "@material-ui/core"
+
+export default class DynamicBreadcrumb extends React.Component {
+
+    render() {
+        var i = 0;
+        return (
+            <Box padding="2em 0em 0em 1em">
+                <Breadcrumbs aria-label="breadcrumb" id="breadcrumbParent">
+                    {this.props.breadcrumbs.map((crumb) => {
+                        return (
+                            <Link color="inherit" href={crumb[1]} id = {"breadcrumb"+i} key={"breadcrumb" + (i++)}>
```

Comment:
why not have the id of the component the same as href name, that way we dont need the i variable